components/clarity-repl/src/repl/interpreter.rs

```diff
@@ -1098,12 +1098,20 @@ impl ClarityInterpreter {
         self.tx_sender.clone()
     }
 
-    pub fn advance_chain_tip(&mut self, count: u32) -> u32 {
-        self.burn_datastore.advance_chain_tip(count);
-        let new_height = self.datastore.advance_chain_tip(count);
+    pub fn advance_burn_chaintip(&mut self, count: u32) -> u32 {
+        let new_height = self.burn_datastore.advance_chain_tip(count);
+        let _ = self.datastore.advance_chain_tip(count);
         self.set_tenure_height();
         new_height
     }
+    pub fn advance_stacks_chaintip(&mut self, count: u32) -> Result<u32, String> {
+        let current_epoch = self.burn_datastore.get_current_epoch();
+        if current_epoch < StacksEpochId::Epoch30 {
+            Err("only burn chain height can be advanced in epoch lower than 3.0".to_string())
```

Comment:
I know this is an idea I suggested instead of the previous error message.
But I just tested in the clarinet console and I don't think the error message is clear for the user. 
it's misleading because in epoch lower than 3.0, both chain are advanced together (not only burn chain height is advanced).
Suggestion: maybe here we should just say that `advance_stacks_chaintip` can't be called in epoch lower than 3.0". But in session.rs, when we actually display an error to the users here https://github.com/hirosystems/clarinet/blob/1e1fc418f5c03dd023cb921e90696ee345e53987/components/clarity-repl/src/repl/session.rs#L842-L843

We should explain that this command should not be called in epoch lower than 3.0 and tell which command to use instead